### PR TITLE
Create HDFS_installation_zh.md

### PR DESCRIPTION
**Pull request by xxisxuxu**
I hereby agree to the terms of the CLA available at: [placeholder]

Changelog category (leave one):

- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Add Chinese doc of Create HDFS_installation.

Detailed description / Documentation draft:

...

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ByConity Documentation](https://github.com/ByConity/ByConity/blob/master/CONTRIBUTING.md) guide first.

### CODE REVIEW

**Comment by xxisxuxu** [comment]
@kevinthfang please review it.

File: docker/executable_wrapper/HDFS_installation_zh.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
+
+# HDFS 安装指南
+在本指南中，我将在 3 台机器上设置 HDFS，其中 1 台机器用于名称节点（namenode），另外 2 台机器用于数据节点（datanode）。 我参考了官方文档 SingleCluster 和 ClusterSetup。 我将安装 HDFS 版本 3.3.4，因此我需要 java-8，因为这是 Hadoop 推荐的 Java 版本。
```
Comment (kevinthfang) [inline docker/executable_wrapper/HDFS_installation_zh.md:3]: 专有名字不用翻译，比如名称节点（namenode），下同

Comment (kevinthfang) [inline docker/executable_wrapper/HDFS_installation_zh.md:3]: “我”改成”这里“比较好，下同，‘我们’可以保留

File: docker/executable_wrapper/HDFS_installation_zh.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        </property>
+        <property>
+                <name>dfs.hosts</name>
+                <value>/root/user_xyz/hdfs/datanodes_list.txt</value>
+        </property>
+
+</configuration>
+```
+
+这就是 namenode 的配置。 现在对于你需要部署 datanode 的那两个节点，创建一个目录来存储 datanode 运行时数据
```
Comment (kevinthfang) [inline docker/executable_wrapper/HDFS_installation_zh.md:76]: 换行

File: docker/executable_wrapper/HDFS_installation_zh.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+我们已经完成配置，现在转到 namenode 机器，转到hadoop分布格式化文件系统并用这个命令启动 namenode
+
+```
+bin/hdfs namenode -format
+bin/hdfs  --daemon start namenode
+```
+
+然后到另外两台datanode机器，到 hadoop 分布，用下面这个命令启动 data node
```
Comment (kevinthfang) [inline docker/executable_wrapper/HDFS_installation_zh.md:100]: ‘分布’改成文件夹下同

**Review by kevinthfang — CHANGES_REQUESTED** [review]

File: docker/executable_wrapper/HDFS_installation_zh.md (hunk 1 above)
Comment (xxisxuxu) [reply docker/executable_wrapper/HDFS_installation_zh.md:3]: > 专有名字不用翻译，比如名称节点（namenode），下同

already edited.

Comment (xxisxuxu) [reply docker/executable_wrapper/HDFS_installation_zh.md:3]: > “我”改成”这里“比较好，下同，‘我们’可以保留

already edited.

File: docker/executable_wrapper/HDFS_installation_zh.md (hunk 2 above)
Comment (xxisxuxu) [reply docker/executable_wrapper/HDFS_installation_zh.md:76]: > 换行

这里指在哪里换行？@kevinthfang

File: docker/executable_wrapper/HDFS_installation_zh.md (hunk 3 above)
Comment (xxisxuxu) [reply docker/executable_wrapper/HDFS_installation_zh.md:100]: > ‘分布’改成文件夹下同

already edited.

**Review by kevinthfang — APPROVED** [review]

**Pull request merged**